pontoon/messaging/static/js/messaging.js

```diff
@@ -162,13 +163,87 @@ $(function () {
     $('#review .message-type .transactional').toggle(isTransactional);
   }
 
+  function loadPanelContent(path) {
+    if (inProgress) {
+      inProgress.abort();
+    }
+
+    const panel = container.find('.right-column');
+    const relative_path = path.split('/messaging/')[1];
+    const menu_path = path === '/messaging/sent/' ? path : '/messaging/';
```

Comment:
Further down there's a check for `if (relative_path === 'sent/')`, why is the check here different?

Reply:
Updated.